### DISCUSSION TITLE
General Doubt's

### DISCUSSION THREAD

**Discussion by Vikash-8090-Yadav** [Vikash-8090-Yadav/Solidity-Pathshala#173] [Q&A] [answered]
## This Discussion is for the General  Talk's and Anyone can ask Doubt  related to Web3  , NFT  , Blockcain , Project Issue and Repo related in this . 
**Let's rock Together**  :rocket: :rocket:

**Comment by Vikash-8090-Yadav** [comment, marked as answer]
#176 OK  let's discus here  , @apoorva1823000 

Can u do one thing , Instead of creating the new contract , can u just modify the  exist one . 
Let me know if u have any  doub't regarding this .

**Reply by apoorva1823000** [reply]
> #176 OK  let's discus here  , @apoorva1823000 
> 
> Can u do one thing , Instead of creating the new contract , can u just modify the  exist one . 
> Let me know if u have any  doub't regarding this . 

Will that be considered a valid contribution ?
And also would you please guide how to do so ?😅

**Reply by Vikash-8090-Yadav** [reply]
Yes  , You just need to clone the repo and open that folder (Level2->Voting project ) in your  local  ans apply all the changes . that  u r saying . 
2. Then do the git  status 
3. and push alll the chnages . 
4. If possible do all this in  a new branch .

**Reply by Vikash-8090-Yadav** [reply]
Feel free to  ask  any doubt  !  :rocket:

**Reply by apoorva1823000** [reply]
Ok let me try that
Also I have coded other smart contracts like one for FIR system, lottery system etc... May I add them too ?
Also Will that be counted as a different contribution ?

**Reply by Vikash-8090-Yadav** [reply]
If ur adding any new smart contract  Firstly check if that already exsist or not 
If exist then raise an issue for the  improvement and modification . 
If not exist then  raise an issue  for that smart contract . 

And if  i talked to your question 
Create an issue for all this . I will  check and if seems  right , then i will assign that to u  .
Don't worry do this in chill mood . :smile:

PS : Feel free to ask anything  and anytime .

**Reply by apoorva1823000** [reply]
Sir I have tried to create pull request with the proposed changes but it is shown in the same pull request as the previous one. Kindly visit that request as the new changes are also visible there

**Reply by Vikash-8090-Yadav** [reply]
Ok wait

**Reply by apoorva1823000** [reply]
I tried to add FIR system using GIT CLI as per the mentioned steps in the readme file but this is the error message
git push -f -u origin FIRBranch
remote: Permission to Vikash-8090-Yadav/Solidity-Pathshala.git denied to apoorva1823000.
fatal: unable to access 'https://github.com/Vikash-8090-Yadav/Solidity-Pathshala.git/': The requested URL returned error: 403

**Reply by Vikash-8090-Yadav** [reply]
@apoorva1823000 have u forked the repo ? Solidity-Pathshala ??

**Reply by apoorva1823000** [reply]
Yup

**Reply by Vikash-8090-Yadav** [reply]
@apoorva1823000 No, bro u haven't forked the repo . Please check once

**Reply by apoorva1823000** [reply]
@Vikash-8090-Yadav 
sorry I had forked it but still it was showing issues but its resolved
I have made changes in the level 2 voting smart contract as well as added a FIR smart contract in the level 1 and have raised a PR
Kindly acknowledge the same
and will these be considered as two contributions or just one ? Just curious

**Reply by Vikash-8090-Yadav** [reply]
Sure  , let me check

**Reply by Vikash-8090-Yadav** [reply]
@apoorva1823000 These are considered as 1 contribution only but I will give u marks a/c yo the issue u have assigned . Intermediate+beginner .
There are some points which u should take care before making pr : 
1. If u have make one PR and that is not yet merge and if u want to raise another pr then and u don't want to create a branch for that then wait for that pr to ber merger. Then raise another PR .
2. If u want to make seperate PR for the two issue one by one then it'als advisable to make   **Two Branch directly from the main anr u can Raise PR from that Banch**
3.Hope this clear ur and in future u ll be able to make OR in two diff forms . Thanks ✌️

**Reply by Vikash-8090-Yadav** [reply]
Am marking this as a answered . For other issue Create a new once along with issue no as I have created in starting .Hope u get this .

**Comment by Syedhussainsaber** [comment]
Sir I am little bit confused, this is mine 1st hackathon so please guide me

**Reply by Saksham-1612** [reply]
Nothing is to panic just see the technology used in project. 
If you know that already just see what you can do in that project. Like in this project you can easily add anything.
For starting  raise an issue to build an easy smart contract and then get it assigned to you just start working on it raise pr

**Reply by Vikash-8090-Yadav** [reply]
@Syedhussainsaber did u  get any work or issue  to work on ?

**Reply by Vikash-8090-Yadav** [reply]
If this issue solved then ,marked this as "answered' to close this

**Comment by Syedhussainsaber** [comment]
Assign me any work

**Reply by Vikash-8090-Yadav** [reply]
OK @Syedhussainsaber . Don't worry i will assign u .

**Reply by Vikash-8090-Yadav** [reply]
can u please tell me your tech stack ? in which u r good and want to work

**Comment by Saksham-1612** [comment]
hey i am confused why i can't comment on my own listed issue
when i tried commenting like quoting something it got closed and i need to reopen it

**Reply by Vikash-8090-Yadav** [reply]
Hey @Saksham-1612  can u please share me  the ss as well as  issue no .

**Reply by Saksham-1612** [reply]
Yeah 
![image](https://user-images.githubusercontent.com/91990962/211171064-15ef8163-80cb-4f21-b675-ee15093900de.png)
so If i will comment on the issue that i only listed it give me two option close as completed or close as not planned.
But i just want to comment I don't want to close the issue.

**Reply by Vikash-8090-Yadav** [reply]
just click on the comment button that is in green colour . u all set to gave comment !

**Comment by Vikash-8090-Yadav** [comment]
@jovinkassi Don't worry i will guide  you about the PR and github  all this by Today eod . !
Chill :smile:
